examples/06-Multiphysics/Maxwell3D_Icepak_2Way_Coupling.py

```diff
@@ -0,0 +1,293 @@
+# Assign excitation
+# ~~~~~~~~~~~~~~~~~
+# Assign coil current, coil consists of 20 turns, total current 10A.
+# Note that each coil turn consists of 24 parallel Litz strands, see above.
+
+no_turns = 20
+coil_current = 10
+m3d.assign_coil(["Coil_terminal"], conductor_number=no_turns, name="Coil_terminal")
+m3d.assign_winding(
+    is_solid=False,
+    current_value=coil_current,
+    name="Winding1",
+)
+
+m3d.add_winding_coils(windingname="Winding1", coil_names=["Coil_terminal"])
+
+###############################################################################
+# Assign mesh operations
+# ~~~~~~~~~~~~~~~~~~~~~~
+# Mesh operations are not necessary in eddy current solver because of auto-adaptive meshing.
+# However, with appropriate mesh operations, less adaptive passes are needed.
+
+m3d.mesh.assign_length_mesh(
+    ["Core"], maxlength=15, meshop_name="Inside_Core", maxel=None
+)
+m3d.mesh.assign_length_mesh(
+    ["Coil"], maxlength=30, meshop_name="Inside_Coil", maxel=None
+)
+
+###############################################################################
+# Set conductivity temperature coefficient
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Set conductivity as a function of temperature. Resistivity increases by 0.393% per K.
+
+cu_resistivity_temp_coefficient = 0.00393
+cu_litz.conductivity.add_thermal_modifier_free_form("1.0/(1.0+{}*(Temp-20))".format(cu_resistivity_temp_coefficient))
+
+###############################################################################
+# Set object temperature and enable feedback
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Set the temperature of the objects to default temperature (22deg C)
+# and enable temperature feedback for two-way coupling.
+
+m3d.modeler.set_objects_temperature(["Coil"])
+
+###############################################################################
+# Assign matrix
+# ~~~~~~~~~~~~~
+# Resistance and inductance calculation.
+
+m3d.assign_matrix(["Winding1"], matrix_name="Matrix1")
+
+###############################################################################
+# Create and analyze simulation setup
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Simulation frequency 150kHz.
+
+setup = m3d.create_setup(setupname="Setup1")
+setup.props["Frequency"] = "150kHz"
+m3d.analyze_setup("Setup1")
+
+###############################################################################
+# Postprocessing
+# ~~~~~~~~~~~~~~
+# Calculate analytical DC resistance and compare it with the simulated coil resistance,
+# print them in the message manager, as well as ohmic loss in coil before temperature feedback.
+
+report = m3d.post.create_report(expressions="Matrix1.R(Winding1,Winding1)")
+solution = report.get_solution_data()
+resistance = solution.data_magnitude()[0]
+
+report_loss = m3d.post.create_report(expressions="StrandedLossAC")
+solution_loss = report_loss.get_solution_data()
+EM_loss = solution_loss.data_magnitude()[0]
```

Comment:
```suggestion
em_loss = solution_loss.data_magnitude()[0]
```
remember to change the other variable instances and re-run again the example to check that everything still work after these changes.